RELEASE_NOTES.md

```diff
@@ -44,6 +49,10 @@ v1.0.0-alpha.xx
   the pointer is never null.
   [#903](https://github.com/OpenAssetIO/OpenAssetIO/issues/903)
 
+- Added default implementations of `getWithRelationship` and
+  `getWithRelationships` that return empty lists, making these methods
+  opt-in for manager implementations.
```

Comment:
[consider][minor] Worth an issue/PR link?

Reply:
I dont think we have a PR that specifically discusses the rationale there in any more detail, do we think its useful to have a PR link to track when it was affected?